debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -108,8 +129,24 @@
             .withDefault(100)
             .withValidation(Field::isInteger);
 
+    public static final Field CONNECTOR_CLASS = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "connector.class")
```

Comment:
Why are these two `Field`s now? Wouldn't it suffice to pass in the values when constructing the history object? I.e. when is there a need to configure this differently than for the connector itself?

Reply:
@gunnarmorling  The history obejct is constructed using `Configuration.getInstance` so it is not possible to pass params to the constructor. We would need to modify the signature of `configure` method but that would mean that we will push thses values to the implementations that does not need them